libs/service-portal/icelandic-names-registry/codegen.yml

```diff
@@ -0,0 +1,18 @@
+schema:
+  - apps/api/src/api.graphql
+documents: libs/service-portal/icelandic-names-registry/src/queries/*.{ts,tsx}
+generates:
+  libs/service-portal/icelandic-names-registry/src/queries/schema.ts:
+    plugins:
+      - typescript
+      - typescript-operations
```

Comment:
This might be copy pasta, but I recommend checking out the typescript-react-apollo plugin: https://www.graphql-code-generator.com/docs/plugins/typescript-react-apollo

It creates hooks for you to use and skip some more boilerplate.